--- safecat-1.11.orig/maildir.sh	2003-04-14 16:03:09.000000000 +0200
+++ safecat-1.11/maildir.sh	2003-04-14 16:03:56.000000000 +0200
@@ -1,4 +1,4 @@
 # Copyright (c) 2000, Len Budney. See COPYING for details.
 
 exec \
-HOME/bin/safecat "$1"/tmp "$1"/new
+/usr/bin/safecat "$1"/tmp "$1"/new
--- ./Makefile.orig	Tue Aug 17 04:50:09 2004
+++ ./Makefile	Wed Sep 22 16:48:02 2004
@@ -31,7 +31,7 @@
 
 auto_home.c: \
 auto-str conf-root
-	./auto-str auto_home `head -1 conf-root` > auto_home.c
+	./auto-str auto_home $(DESTDIR)`head -1 conf-root` > auto_home.c
 
 auto_home.o: \
 compile auto_home.c
--- safecat-1.13/hier.c~	2001-04-14 13:09:54.000000000 +0200
+++ safecat-1.13/hier.c	2006-05-07 22:20:48.498547001 +0300
@@ -7,11 +7,12 @@
   c(auto_home,"bin","safecat",-1,-1,0755);
   c(auto_home,"bin","maildir",-1,-1,0755);
 
-  d(auto_home,"man",-1,-1,02755);
-  d(auto_home,"man/man1",-1,-1,02755);
-  d(auto_home,"man/cat1",-1,-1,02755);
-  c(auto_home,"man/man1","safecat.1",-1,-1,0644);
-  c(auto_home,"man/cat1","safecat.0",-1,-1,0644);
-  c(auto_home,"man/man1","maildir.1",-1,-1,0644);
-  c(auto_home,"man/cat1","maildir.0",-1,-1,0644);
+  d(auto_home,"share/man",-1,-1,02755);
+  d(auto_home,"share/man",-1,-1,02755);
+  d(auto_home,"share/man/man1",-1,-1,02755);
+  d(auto_home,"share/man/cat1",-1,-1,02755);
+  c(auto_home,"share/man/man1","safecat.1",-1,-1,0644);
+  c(auto_home,"share/man/cat1","safecat.0",-1,-1,0644);
+  c(auto_home,"share/man/man1","maildir.1",-1,-1,0644);
+  c(auto_home,"share/man/cat1","maildir.0",-1,-1,0644);
 }
